netty/src/main/java/io/grpc/transport/netty/GrpcSslContexts.java

```diff
@@ -93,7 +111,44 @@ public static SslContextBuilder forServer(
    * an application requires particular settings it should override the options set here.
    */
   public static SslContextBuilder configure(SslContextBuilder builder) {
-    return builder.ciphers(Http2SecurityUtil.CIPHERS, SupportedCipherSuiteFilter.INSTANCE)
-        .applicationProtocolConfig(DEFAULT_APN);
+    return configure(builder, SslContext.defaultServerProvider());
+  }
+
+  /**
+   * Set ciphers and APN appropriate for gRPC. Precisely what is set is permitted to change, so if
+   * an application requires particular settings it should override the options set here.
+   */
+  public static SslContextBuilder configure(SslContextBuilder builder, SslProvider provider) {
+    // The default client/server providers in Netty are the same, so it doesn't matter which
+    // method we call.
+    return builder.sslProvider(provider)
```

Comment:
Oh, I wasn't thinking that we would set the provider, that they would just tell us which provider they were using.

So looking at it now, I'm uneasy about the stability of configure(). It can't distinguish between client and server, must be informed the SslProvider, and we are assuming the client and server providers are the same. Maybe we should make it @ Experimental?